src/v/raft/fundamental.h

```diff
@@ -32,7 +32,7 @@ enum class reply_result : uint8_t {
     success,
     failure,
     group_unavailable,
-    timeout
+    follower_busy
```

Comment:
I see, I guess the timeout case never ends using the `reply_result`, it will be handled in a different path: these codes are only for cases where an RPC was actually received, right?

Reply:
exactly